front_end/src/components/game_room/Player.js

```diff
@@ -120,7 +119,7 @@ export class Player1 extends Component {
       );
     } else if (this.state.if_received_questions && !this.if_submitted_answer) {
       // console.log("we received qustions ! countdown starts now!");
-      return <Countdown date={Date.now() + 10000} renderer={this.renderer} />;
+      return <Countdown date={Date.now() + 3000} renderer={this.renderer} />;
```

Comment:
This is changing the timer to 3 second right? @SalYard

Reply:
Yeah, thats correct